tutorials/fmc/fmc_config.h

```diff
@@ -0,0 +1,27 @@
+#include "libs/adc/api.h"
+#include "libs/timer/api.h"
+
+adc_pin_e adc_pin = ADC0;
+
+void timer0_callback(void);
+
+timer_cfg_s timer0_cfg = {
```

Comment:
Good practice to leave a comment about what the timer configuration is doing since it's pretty hard to tell. Talk about its purpose and how you arrived at 1249 as the number. Same for `timer1_cfg`.